ext/opentelemetry-ext-django/tests/test_middleware.py

```diff
@@ -0,0 +1,13 @@
+from django.test import Client, SimpleTestCase
```

Comment:
This is quite limited testing :) My MR has a lot more coverage (but of course with the different implementation): https://github.com/open-telemetry/opentelemetry-python/pull/571/files?file-filters%5B%5D=.ini&file-filters%5B%5D=.py#diff-ef5c6d230cc8cf526b64f232d2ed0470

Reply:
Added more testing